sql-statements/sql-statement-select.md

```diff
@@ -159,6 +161,63 @@ mysql> SELECT AVG(s_quantity), COUNT(s_quantity) FROM stock;
 
 The above example uses data generated with `tiup bench tpcc prepare`. The first query shows the use of `TABLESAMPLE`.
 
+### SELECT ... INTO OUTFILE
+
+`SELECT ... INTO OUTFILE` is used to write the result of a query to a file. Column and line terminators can be specified to produce a specific output format.
+
+You can use the `Fields` and `Lines` parameters to specify how to handle the data format.
```

Comment:
Isn't this duplicating the previous sentence?

Reply:
OK, I merged these 2 lines.